src/image.js

```diff
@@ -34,7 +35,7 @@ export default class Image extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ ImageEditing, Widget, ImageTextAlternative ];
+		return [ ImageEditing, Widget, ImageTextAlternative, ImageResize ];
```

Comment:
Just like `ImageCaption` or `ImageStyle`, this should be an optional plugin. So, it should not be listed here.